besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1489,6 +1429,7 @@ public void run() {
 
       // Need to create vertx after cmdline has been parsed, such that metricsSystem is configurable
       vertx = createVertx(createVertxOptions(metricsSystem.get()));
+      miningParameters = getMiningParameters();
```

Comment:
Is there a reason why this needs to happen here? Inside of the Run method? Can we move it to the configure method?

Reply:
removed since call is not needed anymore